src/languageProvider/locales/en_US.json

```diff
@@ -57,12 +58,20 @@
   "create.validResultSetStart": "Must be greater than or equal to Betting End Time",
   "depositDialog.title": "QTUM/BOT Deposit Address",
   "eventHistory.viewEvent": "View Event",
+  "eventInfo.creator": "CREATOR",
+  "eventInfo.endDate": "ENDING DATE",
+  "eventInfo.fund": "FUNDING",
+  "eventInfo.predictionFund": "PREDICTION FUNDING",
+  "eventInfo.resultSetter": "RESULT SETTER",
+  "eventInfo.voteVolumn": "VOTING VOLUME",
   "language.select": "中文",
   "myBalances.addressCopied": "Address copied",
   "myBalances.deposit": "Deposit",
   "myBalances.myBalance": "My Balance",
   "myBalances.withdraw": "Withdraw",
   "navBar.activities": "My Activities",
+  "navbar.botCourt": "BOT Court",
+  "navbar.qtumPredict": "QTUM Prediction",
```

Comment:
we should use the full string name if it is short enough. `navBar.qtumPrediction`. if its too long then we can use abbreviations.